File: Unidad 2/Meta 2.1/Java/SistemaBiologico.java
public class SistemaBiologico {
    private String nombre;

    public SistemaBiologico(String nombre) {
        this.nombre = nombre;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    @Override
    public String toString() {
        return "SistemaBiologico{" + "nombre=" + nombre + '}';
    }
}

File: Unidad 2/Meta 2.1/Python/SistemaBiologico.py
#!/usr/bin/python
# -*- coding: UTF-8 -*-
from typing import List

class SistemaBiologico(object):
	def __init__(self, nombre: str):
		self.nombre = nombre

	def setNombre(self, nombre: str):
		self.nombre = nombre

	def getNombre(self) -> str:
		return self.nombre	

	def __str__(self):
		return "SistemaBiologico{" + "nombre=" + self.nombre + '}'	
